changelog.d/7704.misc

```diff
@@ -0,0 +1 @@
+Replace more uses of `six` with native Python 3 equivalents.
```

Comment:
You can credit yourself with any changes if you'd like (add "Contributed by <your name>." at the end).